data/templates/css/wikimedia.scss

```diff
@@ -635,4 +692,20 @@ $spacing-elements: 30px;
     #toggle:checked ~ .toggle_label:after {
         content: "▼";
     }
+
+    #bodycontents #inside-content .thumb,
+    #bodycontents #inside-content .tright,
+    .thumb .thumbcaption{
+        margin-bottom: 0px;
+    }
+
+    #bodycontents #inside-content ul,
```

Comment:
This can be
```scss
#bodycontents #inside-content {
    ul, ol, ul li, ul ul, ol li {
        width: auto;
        margin-left: 0;
    }
}
```
although I don't think `ul ul` needs to be mentioned specifically, and I think it could just be `ul, ol, li`.